### PR TITLE
New response format for Events via graph api

### PR DESCRIPTION
**Pull request by ivankocienski**
Implements #1049

### CODE REVIEW

File: app/graphql/types/event_type.rb [hunk 1]
```diff
@@ -1,13 +1,15 @@
 module Types
   class EventType < Types::BaseObject
 
-    description 'An Event that is run by a Parter'
-
     field :id, ID, null: false
-    field :description, String
+    field :name, String, method: :summary
```
Comment (kimadactyl) [inline app/graphql/types/event_type.rb:5]: Maybe just add a comment here like

# Summary and name are aliases, this is left as a convenience for people used to iCal format

File: app/graphql/types/query_type.rb [hunk 2]
```diff
@@ -33,7 +33,7 @@ def event(id:)
     end
 
     def event_connection(**args)
-      Event.all
+      Event.sort_by_time.all
```
Comment (kimadactyl) [inline app/graphql/types/query_type.rb:36]: Does this filter from `Time.now`?

Comment (ivankocienski) [reply app/graphql/types/query_type.rb:36]: This orders by `dtstart` descending.

Comment (kimadactyl) [reply app/graphql/types/query_type.rb:36]: I think there's no point returning events that are in the past so maybe should update the filter? (assume you mean ascending too)

Comment (ivankocienski) [reply app/graphql/types/query_type.rb:36]: as i understand- the events are returned in the same paginated way as partners, just ordered by dtstart so if you have an offset of 100 then all the events will be returned after 100 ordered by dtstart. there is currently no way to specify a 'from date' field in the query

**Review by kimadactyl — APPROVED** [review]

**Pull request merged**